src/front-end/src/app/shared/components/dog-card/dog-card.component.scss

```diff
@@ -1,27 +1,31 @@
-$card-height: 400px;
-$card-width: 300px;
+$card-height: 500px;
+// $card-max-width: 300px;
 
-:host {
-    display: flex;
-    justify-content: center;
+// :host {
+//     display: flex;
+//     justify-content: center;
+// }
```

Comment:
Commented code again.
Don't worry, it's fine to delete code - it will be available in the history.
The general rule of source control systems "don't comment - remove"